specification/network/resource-manager/Microsoft.Network/preview/2024-10-01-preview/networkManagerBillingMeter.json

```diff
@@ -0,0 +1,200 @@
+        ],
+        "operationId": "NetworkManagerBillingMeter_Get",
+        "description": "Gets the specified Network Manager's Billing Meter.",
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the Billing Meter for given network manager.",
+            "schema": {
+              "$ref": "#/definitions/NetworkManagerBillingMeter"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "NetworkManagerBillingMeterGet": {
+            "$ref": "./examples/NetworkManagerBillingMeterGet.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "NetworkManagerBillingMeter"
+        ],
+        "operationId": "NetworkManagerBillingMeter_Patch",
+        "description": "Patch NetworkManagerBillingMeter.",
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PatchObject"
+            },
+            "description": "Parameters supplied to specify Billing meter the network manager uses. Note - only VnetLevel is supported for Update operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updated - Returns information about the NetworkManager BillingMeter.",
+            "schema": {
+              "$ref": "#/definitions/NetworkManagerBillingMeter"
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "NetworkManagesPatch": {
+            "$ref": "./examples/NetworkManagerBillingMeterPatch.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NetworkManagerBillingMeter": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NetworkManagerBillingMeterProperties",
+          "description": "The NetworkManagerBillingMeter properties"
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system metadata related to this resource.",
+          "$ref": "./network.json#/definitions/SystemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/ChildResource"
```

Comment:
do you need all of the child resource props?

Reply:
yes